commit a1a77f34c16b03b88af62fdc36996469960c50a8
Author: Robert Bogart <robert.bogart@joyent.com>
Date:   2019-09-16T19:08:22+00:00 (3 weeks ago)
    
    MANTA-4553 Allow rebalancer agent to listen on INADDR_ANY for now

diff --git a/src/main.rs b/src/main.rs
index 486507e..8d7fd65 100644
--- a/src/main.rs
+++ b/src/main.rs
@@ -23,7 +23,12 @@ fn main() -> Result<(), Error> {
         SubCommand::Server => Ok(()),
         SubCommand::DoJob(job) => job.run(),
         SubCommand::Agent => {
-            Agent::run("127.0.0.1:7878");
+            // We should only be using 0.0.0.0 (INADDR_ANY) temporarily.  In
+            // production we will be supply an ip address that is obtained from
+            // the config file that we process which will dictate the network
+            // on which to listen.  It is worth mentioning that this will likely
+            // also be the case for the agent port.
+            Agent::run("0.0.0.0:7878");
             Ok(())
         }
     }
